Gemfile

```diff
@@ -67,7 +67,8 @@ else
 end
 
 gem 'sass', '3.4.9'
-gem 'sass-rails'
+gem 'sass-rails' # required by bootstrap-sass, but not listed as a dependency of it
+gem 'sassc-rails', github: 'bolandrm/sassc-rails', branch: 'master'
```

Comment:
Good spot @edds.

We'd have to `bundle update` to change the version, and the lock file would change and need to be checked in to take effect. In theory that means both the committer and the reviewer would be encouraged to ask "why is this?", but it would definitely be better to specify a tag or a sha here.